src/styles/common.scss

```diff
@@ -80,15 +80,14 @@
 
   .bubble.right {
       border-radius: 13px 2px 13px 13px;
-      box-shadow: 0 -1px 12px 0 rgba(0, 0, 0, 0.1);
-      background-color: $vue-light-blip;
-      color: $vue-white;
+      background-color: $color-surface-3;
```

Comment:
Should the `background color` be different between `.bubble.right` and `.bubble.left`?

Reply:
Yes it should. It was a definition of the design team.